projects/storefrontlib/src/lib/cms/facade/cms.service.ts

```diff
@@ -19,7 +20,7 @@ export class CmsService {
     private defaultPageService: DefaultPageService
   ) {}
 
-  getComponentData(uid: string): Observable<any> {
+  getComponentData(uid: string): Observable<Component> {
```

Comment:
This one should be: `getComponentData<T extends Component>(uid: string): Observable<T> {`